content/it/case-studies/OWNERS

```diff
@@ -0,0 +1,8 @@
+# Owned by Kubernetes Blog reviewers.
```

Comment:
Can you remove this file? This is not needed right now and can be easily added at a later point.